src/raftstore/store/peer.rs

```diff
@@ -227,6 +228,7 @@ pub struct Peer {
 
     pub written_bytes: u64,
     pub written_keys: u64,
+    pub latestn_written_bytes: LatestN,
```

Comment:
I don't think so, in that way may can not catch the real hot region. Use the sum of latest N works like monitor CPU load.